SPECS/csi-driver-lvm/csi-driver-lvm.spec

```diff
@@ -20,9 +20,23 @@ Source0:        https://github.com/metal-stack/%{name}/archive/refs/tags/v%{vers
 #           -cf %%{name}-%%{version}-govendor.tar.gz vendor
 Source1:        %{name}-%{version}-govendor.tar.gz
 BuildRequires:  golang
+Requires:       %{name}-csi-lvmplugin-provisioner
```

Comment:
Curious, why split the package into two RPMS if they must both be installed together in order to function?

Reply:
For organization purposes, we will leverage these two packages to create docker images for golden container, so in the final, there will be two docker images pushed to the remote.

Reply:
Not sure I understand, will the two docker containers be shipped with these packages separately? My impression is that one will not function without the other, correct?

Reply:
Yes, that is right.

Reply:
@Camelron, we are following a similar pattern in [cert-manager spec](https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/cert-manager/cert-manager.spec). Unless there's a difference guidance, it makes sense to keep components of the same opensource project that gets deployed together in the same spec.